### ISSUE TITLE
[💡 Feature Request]: 

### ISSUE THREAD

**Issue by ThemihanWeb** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#553]
### What's the Problem?

When I tell open MS word or open my browser in my windows pc, it tells I can't open apps on this device. Google assistant desktop client doesn't have full control of my windows pc.
![image](https://user-images.githubusercontent.com/92494550/137349691-6014c16c-8005-41d4-8e74-51a929686594.png)
![image](https://user-images.githubusercontent.com/92494550/137351753-dd21a765-e18c-4439-89a0-ecdf7d5a20a7.png)

### What's your solution or idea?

I would like google assistant to be able to open apps on windows pc.
I would like google assistant to shut down or sign out or sleep the windows pc.
I would like to replace google assistant instead of cortana

### Alternatives Considered

.

### App Version

Built from source

### OS Version

Windows 10 Version 21H1 (OS Build 19043. 1237)

### Additional context

.

### Pre-submission Checklist

- [X] I have checked [the issue tracker](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/issues?q=is%3Aissue+label%3A%22%E2%9C%A8+enhancement%22+) and didn't find any duplicate issue
- [X] This issue only covers one particular feature request

**Comment by im-coder-lg** [comment]
This won't work since GAUDC is unofficial, so it is not integrated deeply inside the computer. This is a duplicate of an issue.

**Comment by im-coder-lg** [comment]
> we can integrate this with the help of third party apps maybe

No. You n red a seperate module to do that. If you know one, do tell us here!
